Monika After Story/game/zz_music_selector.rpy

```diff
@@ -93,7 +93,10 @@ init -1 python in songs:
             music_choices.append(("Your Reality","bgm/credits.ogg"))
             music_choices.append(("I Still Love You","bgm/monika-end.ogg"))
             music_choices.append(("Okay, Everyone! (Monika)","<loop 4.444>bgm/5_monika.ogg"))
-
+            store.seen_event
```

Comment:
alright dude, `seen_event` is still a function, so you need to call it like one as you did before. the only difference is that it needs to have `store` in front of it because of scope. code inside a `store` isnt aware of code outside of its `store` unless you tell it where to find it. (prepending with `store`). Look up scope if you need understanding.